.circleci/config.yml

```diff
@@ -6,10 +6,9 @@ orbs:
 parameters:
   image_suffix:
     type: string
-    default: "-v2021_10_27"
+    default: '-dev-202112030707'
```

Comment:
Don't forget to change this after we have landed the CI changes and pushed released images.